ts/src/main/scala/ch/epfl/ts/brokers/ExampleBroker.scala

```diff
@@ -0,0 +1,129 @@
+package ch.epfl.ts.brokers
+
+import akka.actor.{ActorLogging, Props, ActorRef}
+import akka.pattern.ask
+import akka.util.Timeout
+import ch.epfl.ts.component.Component
+import ch.epfl.ts.data._
+import ch.epfl.ts.engine._
+import ch.epfl.ts.engine.GetWalletFunds
+import ch.epfl.ts.engine.WalletConfirm
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import ch.epfl.ts.data.{Order, MarketBidOrder}
+import ch.epfl.ts.engine.WalletConfirm
+import ch.epfl.ts.data.Register
+import ch.epfl.ts.engine.FundWallet
+import ch.epfl.ts.engine.WalletFunds
+import ch.epfl.ts.engine.GetWalletFunds
+import ch.epfl.ts.engine.WalletInsufficient
+import ch.epfl.ts.data.ConfirmRegistration
+import ch.epfl.ts.data.Currency._
+import ch.epfl.ts.engine.WalletConfirm
+import ch.epfl.ts.data.Register
+import ch.epfl.ts.engine.FundWallet
+import scala.Some
+import ch.epfl.ts.engine.WalletFunds
+import ch.epfl.ts.engine.GetWalletFunds
+import ch.epfl.ts.engine.WalletInsufficient
+import ch.epfl.ts.data.Currency
+import ch.epfl.ts.data.ConfirmRegistration
+
+/**
+ * Created by sygi on 03.04.15.
+ */
+class ExampleBroker extends Component with ActorLogging { //TODO(sygi): println -> log.debug
```

Comment:
Since it looks simple and wouldn't add code, could you take care of this TODO in this PR?